pen/views.py

```diff
@@ -0,0 +1,75 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.shortcuts import render, redirect
+from .models import Memo
+from django.contrib.auth import authenticate, logout
+from django.contrib.auth import login
+from django.views.decorators.http import require_http_methods
+
+
+def memo(request):
+    return render(
+        request,
+        'pen/memo.html'
+    )
+
+
+@require_http_methods(["POST"])
+def add_new_memo(request):
+    memo = Memo()
+    memo.body = request.POST["memo_body"]
```

Comment:
Tutaj chcemy nie tylko, żeby została zapisana treść notatki, ale też użytkownik ktróry tę notatkę sporządził, czyli nie uzupełnić trzeba dla Memo nie tylko pole memo.body ale też memo.author. Dodatkowo, jeżeli dopuszczamy istnienie tylko jednej notatki dla jednego użytkownika, to trzeba będzie najpierw sprawdzić czy notatka dla danego użytkownik istnieje, Jeżeli istnieje to wziąć ją i zmodyfikować. Dopiero jeżeli nie istnieje to stworzyć nową, czyli to co tu robisz.

Reply:
Kod został poprawiony w taki sposób, aby nie tylko zapisywał treść notatki, ale tez użytkownika, który tę notatkę sporządził, uzupełniono pole memo.author. Dopuszczamy istnienie tylko jednej notatki dla jednego użytkownika. W widokach najpierw sprawdzamy czy notatka dla danego użytkownika istnieje. Jeżeli istnieje to notatka jest modyfikowana. Jeżeli nie istnieje to tworzymy nową notatkę.